src/cluster.h

```diff
@@ -288,9 +286,19 @@ typedef struct {
                                             master is up. */
 
 /* ---------------------- API exported outside cluster.c -------------------- */
+void clusterInit(void);
+void clusterCron(void);
+void clusterBeforeSleep(void);
 clusterNode *getNodeByQuery(client *c, struct redisCommand *cmd, robj **argv, int argc, int *hashslot, int *ask);
+clusterNode *clusterLookupNode(const char *name);
 int clusterRedirectBlockedClientIfNeeded(client *c);
 void clusterRedirectClient(client *c, clusterNode *n, int hashslot, int error_code);
+void migrateCloseTimedoutSockets(void);
```

Comment:
This is only run in serverCron, but it could just be moved to the clusterCron.

Reply:
Sure -- will do

Reply:
(it would mean calling clusterCron every iteration and using run_with_period internally there -- but I think that's a good thing)

Reply:
Actually - I take this back.
MIGRATE can be used regardless of OSS cluster being enabled or not -- so calling its cleanup from within the clusterCron makes less sense.
Eventually we should probably move all the migrate code out of cluster.c since it's orthogonal to the clustering implementation.... (migrate.[c.h]?)